app/matplotlib.ipynb

```diff
@@ -47,21 +45,42 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
    "metadata": {
     "collapsed": true,
     "ExecuteTime": {
-     "end_time": "2023-07-07T12:36:31.669207Z",
-     "start_time": "2023-07-07T12:36:31.661449Z"
+     "end_time": "2024-12-30T15:19:46.601185Z",
+     "start_time": "2024-12-30T15:19:46.003308Z"
     }
    },
-   "outputs": [],
    "source": [
     "years = [2015, 2016, 2017, 2018, 2019]\n",
     "population = [100, 120, 150, 180, 200]\n",
     "\n",
-    "# write your code here"
-   ]
+    "plt.plot(years, population)"
```

Comment:
The line plot is missing a title. You should add a title to the plot using `plt.title('Your Title Here')`.